propagation_test.go

```diff
@@ -83,7 +84,7 @@ func TestSpanPropagator(t *testing.T) {
 	exp.Start = time.Time{}.Add(1)
 
 	for i, sp := range spans {
-		if a, e := sp.ParentSpanID, exp.ParentSpanID; a != e {
+		if a, e := sp.ParentSpanID, exp.SpanID; a != e {
```

Comment:
I thought this should be `exp.SpanID`... it's also the value that gets the test to pass. :)  Let me know if I misunderstand something about Zipkin here.